user/contactmethod/store.go

```diff
@@ -249,6 +249,12 @@ func (db *DB) FindOne(ctx context.Context, id string) (*ContactMethod, error) {
 	if err != nil {
 		return nil, err
 	}
+	num, err := libphonenumber.Parse(c.Value, "")
```

Comment:
Since this is being used more than once, I think we should put them in a separate function.

Reply:
Yeah, when we move the struct field to the `FormattedValue` method, it should happen there.

Make sure to add a switch on type so we only format phone types

Something like
```go
func (cm *ContactMethod) FormattedValue() (string, error) {
  switch cm.Type {
  case TypeSMS, TypeVoice:
   // libphonenumber
  }
  return cm.Value, nil
}
```